scout/server/blueprints/cases/templates/cases/case.html

```diff
@@ -76,19 +76,22 @@ <h4 class="card-header">Case: {{case.display_name}}</h4>
       </div>
 
       <div class="row">
+        {% if case.track != 'cancer' %}
         <div class="col-md-4">
           {% if case.madeline_info and case.individuals|length > 1 %}
             {{ pedigree_panel(case) }}
           {% else %}
             <p>No pedigree picture available.</p>
           {% endif %}
         </div>
+        {% endif %}
 
-        <div class="col-md-8">
```

Comment:
Replacing `col-md-8` with `col` to make the lines expand the whole column. Are we using the other screen layouts? (small, medium, large, XL)

Reply:
I don't know about this. Maybe @northwestwitch can chime in?

Reply:
Agreed - @northwestwitch is the guru there! One way to think about it is that the sizes refer to the smallest size intended. And scout is generally not designed for the very small devices.

Some docs here https://getbootstrap.com/docs/4.4/layout/grid and more generally here https://getbootstrap.com/docs/4.4/layout/overview/#responsive.

Reply:
I'll take a look as soon as I finish reviewing another PR!

Reply:
Haha definitely not the guru, I just google all the time! 😄 
We generally use the layouts mentioned by @dnil. It works already with the code that you wrote, but if you want to be super sure you could write col-12 to force the column to stretch over the entire line!